feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.module.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { I18nModule, provideDefaultConfig } from '@spartacus/core';
+import { ItemCounterModule, KeyboardFocusModule } from '@spartacus/storefront';
+import { defaultConfiguratorUISettingsConfig } from '../../config/default-configurator-ui-settings.config';
+import { ConfiguratorAttributeQuantityComponent } from './configurator-attribute-quantity.component';
+
+@NgModule({
+  declarations: [ConfiguratorAttributeQuantityComponent],
+  entryComponents: [ConfiguratorAttributeQuantityComponent],
+  exports: [ConfiguratorAttributeQuantityComponent],
+  imports: [CommonModule, I18nModule, ItemCounterModule, KeyboardFocusModule],
```

Comment:
is `KeyboardFocusModule` required?

Reply:
Picked

Reply:
KeyboardFocusModule is required. Without it the navigation with keyboard would not work on quantity input field.